File: LAB/LAB1/iterativo.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>


int contarM (char s[]){

int contador=0;

    for(int x=0; s[x]!='\0'; x++){

        if(s[x]>='A'  && s[x]<='Z'){
             contador ++;
        }
      }



return contador;
}

int main(){

char texto[1000];

scanf("%[^\n]", texto);

while(  strcmp("FIM", texto) !=0){


    
printf("%d\n", contarM(texto));
scanf(" %[^\n]", texto);
}


    return 0;
}

File: LAB/iterativo.java
import java.util.Scanner;
public class iterativo{

public static int contarM(String s){ 


    int contador=0;

    for(int x=0; x<s.length(); x++){

        if(s.charAt(x)>='A'  && s.charAt(x)<='Z'){
             contador ++;
        }
      }

return contador;

}

public static void main(String[] args){

String texto;

Scanner ler =new Scanner(System.in);
    
texto= ler.nextLine();

while(!texto.equals("FIM")){


System.out.println(contarM(texto) );
 texto = ler.nextLine();
}


ler.close();
}
}